adapters/cryptopunks2/index.js

```diff
@@ -0,0 +1,123 @@
+require("dotenv").config();
+
+const moment = require("moment");
+const BigNumber = require("bignumber.js");
+const Ethereum = require("../../sdk/EVMC");
+const axios = require("axios");
+const URL = "http://nft-sales-service.dappradar.com/open-source";
+const KEY = process.env.DAPPRADAR_API_KEY;
+const path = require("path");
+
+class CRYPTOPUNKS2 {
+
+    constructor() {
+        this.name = "cryptopunks2";
+        this.symbol = "PUNKS2";
+        this.token = "0xc02d332AbC7f9E755e2b1EB56f6aE21A7Da4B7AD";
+        this.protocol = "polygon";
```

Comment:
it should be matic

Reply:
@Arnaslt have updated to use matic here but the getSymbol API method (Above) still fails for our token.

Reply:
KFMWWHYDUD8BWZ8UQ7TFYSGHZZ5GX1G3D9